Resources/Prototypes/Entities/Clothing/Shoes/boots.yml

```diff
@@ -87,5 +87,95 @@
     sprite: Clothing/Shoes/Boots/winterboots.rsi
   - type: Clothing
     sprite: Clothing/Shoes/Boots/winterboots.rsi
-  - type: TemperatureProtection
-    coefficient: 0.2
+
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterCargo
+  name: cargo winter boots
+  description: Fluffy boots to help survive even the coldest of winters.
```

Comment:
move this description to the parent prototype (same for the rest, unless you want to make new descriptions for one or another)

Reply:
The problem is that the web boots are also connected to this prototype, isn't it easier to just leave it that way than to create a new parent for the web boots?

Reply:
I'm not asking you to make a new parent prototype, just inherit it like you are doing right now but override the description

Reply:
Just move the description to the base prototype and then override the description in the web boots prototype, also remember to remove the description of ClothingShoesBootsWinter once you have one in the base prototype